app/src/test/java/com/litewallet/tools/util/BRConstantsTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.litewallet.tools.util
+
+import com.breadwallet.tools.util.BRConstants
+import org.junit.Assert
+import org.junit.Assert.assertSame
+import org.junit.Test
+
+//TODO: migrate from [com.litewallet.analytics.ConstantsTests]
+class BRConstantsTest {
```

Comment:
need to fill, please let me know if have questions

Reply:
@andhikayuana may check in the values onto this PR